pkg/discovery/agent.go

```diff
@@ -44,18 +41,22 @@ func NewAgent(cfg *config.AgentConfig, client anypoint.Client, sm subscription.S
 		publishAPI:  coreAgent.PublishAPI,
 	}
 
+	assetCache := cache.GetCache()
```

Comment:
GetCache returns the global cache, if that is what you want.  If you would like a separate cache you can call New.

This is fine for your current usage, just a note for the future.

Reply:
That sounds good, we will look into that.